scripts/md2html/md2html.js

```diff
@@ -266,34 +266,51 @@ for (let l in lines) {
     if (!inCodeBlock) {
 
         // minor fixups to get RFC links to work properly
-        if (line.indexOf('RFC [')>=0) {
-            line = line.replace('RFC [','[RFC');
-        }
+        line = line.replace('RFC [','[RFC');
         line = line.replace('[Authorization header as defined in ','Authorization header as defined in [');
+        line = line.replace('[JSON Pointer]','JSON Pointer [RFC6901]'); // only in 2.0.md
+
+        //TODO: more "hidden" RFC references in older specs, for example
+        // [media type range](https://tools.ietf.org/html/rfc7231#appendix-D)
+        // [ABNF](https://tools.ietf.org/html/rfc5234)
+
+        //TODO: unconventional references to RFCs in 3.0.4 and 3.1.1, for example
+        // [RFC3986 §5.1.2 – 5.1.4](https://tools.ietf.org/html/rfc3986#section-5.1.2)
+        // RFC6570 [mentions](https://www.rfc-editor.org/rfc/rfc6570.html#section-2.4.2)
+        // [are not](https://datatracker.ietf.org/doc/html/rfc3986#appendix-A)
+        // [special behavior](https://www.rfc-editor.org/rfc/rfc1866#section-8.2.1)
+        // [RFC6570 considers to be _undefined_](https://datatracker.ietf.org/doc/html/rfc6570#section-2.3)
```

Comment:
I'm entirely fine with making these more normal rather than working around them in the build script.